e2e/scaffolding/extrinsicHelpers.ts

```diff
@@ -264,6 +264,10 @@ export class Extrinsic<N = unknown, T extends ISubmittableResult = ISubmittableR
   public async fundAndSendUnsigned(source: KeyringPair) {
     await this.fundOperation(source);
     log('Fund and Send', `Fund Source: ${getUnifiedAddress(source)}`);
+    return this.sendUnsigned();
```

Comment:
Deleted code block; now that `sendUnsigned` is doing what this block used to do.